sdk/python/kfp/dsl/_container_op.py

```diff
@@ -76,9 +76,9 @@ def __init__(self, name: str, image: str, command: str=None, arguments: str=None
       self.inputs += self.argument_inputs
 
     self.outputs = {}
-    if file_outputs:
-      self.outputs = {name: _pipeline_param.PipelineParam(name, op_name=self.name)
-          for name in file_outputs.keys()}
+    if self.file_outputs:
```

Comment:
Does this change anything?

Reply:
Done. Must've changed it in the past changes.